### ISSUE TITLE
ActiveField with bsVer == 3 does reset the $config

### ISSUE THREAD

**Issue by abriosi-benassi** [kartik-v/yii2-widget-activeform#133]
The ActiveField on line 505 does reset the default $config if bsVer is 3

Solution is either to simply change the variuos options (like in bsVer 4) or set all the options

To reproduce the problem create an INLINE form with bs 3 and see that the various fields miss the form-group class.

If needed I might provide a pull-request, but dunno which approach is better:

```
$config['hintOptions'] = ...
$config['errorOptions'] = ...
```

or simply set the 
```
$config[
  all options go here.
];
```

**Comment by abriosi-benassi** [comment]
Ok, this seems fixed in master (related to #131)

**Comment by williamyiu** [comment]
it is missing labelOptions as well.

I had to add a line in 528 with the following:

			$config['labelOptions'] = ['class' => 'control-label ' . $cssClasses['label']];
